ResearchKit/Charts/ORKFloatRange.h

```diff
@@ -37,52 +38,63 @@
 NS_ASSUME_NONNULL_BEGIN
 
 /**
- The `ORKRangedPoint` class represents a ranged point used in a graph plot.
+ The `ORKFloatRange` class represents a float range. It can be used in graph chart plots to draw a
+ value ranges.
  */
 ORK_CLASS_AVAILABLE
-@interface ORKRangedPoint : NSObject
+@interface ORKFloatRange : NSObject
```

Comment:
If `float` in name indicates that this class carries float values, and we don't plan to introduce `ORKIntegerRange`.
Maybe `ORKValueRange` is a slightly better name?